src/cli/commands/_color-key-for-versions.js

```diff
@@ -0,0 +1,24 @@
+/* @flow */
+
+import type {FormatKeys} from '../../reporters/format.js';
+import semver from 'semver';
+
+const versionColorScheme = {
+  major: 'red',
+  premajor: 'red',
+  minor: 'yellow',
+  preminor: 'yellow',
+  patch: 'green',
+  prepatch: 'green',
+  prerelease: 'red',
+  unchanged: 'white',
+  unknown: 'red',
+};
+
+export default function(from: string, to: string): FormatKeys {
+  const [validFrom, validTo] = [semver.valid(from), semver.valid(to)];
```

Comment:
Destructuring looks nice here but you now add the overhead of creating an array and destructuring it only for aesthetics. I'd just do two separate `const`s as follows:
```
const validFrom = semver.valid(from);
const validTo = semver.valid(to);
```